test/cucumber/Conferencia.feature

```diff
@@ -1,8 +1,8 @@
 @i9n
 Feature: conferencia
   As a member of a research group
-  I want to add, remove and modify conferencias I have published
-  so that I can generate web pages and reports containing these conferencias
+  So that I can add, remove and modify conferencias I had published
+  I want to generate web pages and reports containing these conferencias
```

Comment:
O formato da feature não é adequado. Existem 2 formatos para escrita de features (vide capítulo 4 do livro):
1 - In order to [meet some goal]... as a [role]... I want [a feature]...
2 - As a [role]... I want [a feature]... so that I can [meet some goal]...
O primeiro formato enfatiza o objetivo de negócio, enquanto que o segundo enfatiza o problema (a necessidade de um stakeholder).